src/components/elements/section/section.tsx

```diff
@@ -2,34 +2,49 @@ import { V2_Layout } from "@lifesg/react-design-system/v2_layout";
 import { Wrapper } from "../wrapper";
 import { Contained, GridWrapper } from "./section.styles";
 import { ISectionProps } from "./types";
+import { Layout } from "@lifesg/react-design-system/layout";
 
 export const Section = (props: ISectionProps) => {
 	// =============================================================================
 	// CONST, STATE, REF
 	// =============================================================================
 	const {
-		sectionSchema: { children, layoutType },
+		sectionSchema: {
+			children,
+			layoutType,
+			customOptions = {
+				gridType: "v2",
+				contentType: "v2",
+			},
+		},
 		...otherProps
 	} = props;
 
 	// =============================================================================
 	// RENDER FUNCTIONS
 	// =============================================================================
-	const renderInGrid = () => (
-		<V2_Layout.Section>
-			<GridWrapper type="grid">
-				<Wrapper {...otherProps}>{children}</Wrapper>
-			</GridWrapper>
-		</V2_Layout.Section>
-	);
+	const renderInGrid = () => {
+		const Section = customOptions.gridType === "v3" ? Layout.Section : V2_Layout.Section;
+		const Container = customOptions.gridType === "v3" ? Layout.Container : V2_Layout.Container;
```

Comment:
rename variables to avoid shadowing component name

```suggestion
		const LayoutSection = customOptions.gridType === "v3" ? Layout.Section : V2_Layout.Section;
		const LayoutContainer = customOptions.gridType === "v3" ? Layout.Container : V2_Layout.Container;
```